websites/V/VtuberWiki/presence.ts

```diff
@@ -0,0 +1,118 @@
+type StaticPagesType = Record<string, PresenceData>;
+
+const presence = new Presence({
+		clientId: "1164189020922843257",
+		appMode: true,
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	staticPages: StaticPagesType = {
+		"/": {
+			details: "Viewing the homepage",
+			buttons: [{ label: "View Website", url: "https://wiki.hylia.dev" }],
```

Comment:
Buttons that redirect to main pages are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets).
```suggestion
```